vector/src/main/java/im/vector/app/features/location/MapState.kt

```diff
@@ -18,15 +18,9 @@ package im.vector.app.features.location
 
 import android.graphics.drawable.Drawable
 
-interface VectorMapView {
```

Comment:
This interface was implemented but not really used as a type

Reply:
Yes, I tried to be flexible in case of changing the map library but has problems with epoxy.